### DISCUSSION TITLE
add forum sections for other consoles

### DISCUSSION THREAD

**Discussion by meleu** [RetroAchievements/RAWeb#650] [Ideas]
currently we need sections for
- Master System
- Nintendo 64
- Atari Lynx
- Neo Geo Pocket
- Atari 2600
- Arcade

In the future we'll need a section for:
- PlayStation

And maybe there are more to come!

**Comment by luchaos** [comment]
This could be done within a (new) consoles management area.
Currently there is no area for admins/super-admins to manage consoles.
Should allow to reference a forum or create one if not existing yet.
